tasks/managed/create-pyxis-image/create-pyxis-image.yaml

```diff
@@ -4,7 +4,7 @@ kind: Task
 metadata:
   name: create-pyxis-image
   labels:
-    app.kubernetes.io/version: "4.0.0"
+    app.kubernetes.io/version: "5.0.0"
```

Comment:
Although I don't understand exactly the impact of this change, I am not sure that it really needs a full version bump. It looks like just bumping the patch would be necessary, or at most the minor. WDYT?

Reply:
It really depends what we call a breaking change. Either the default is changing, or the whole option name is changing. While it's not a breaking change in the traditional sense that task parameters or results are being changed, it is a potentially breaking change for users of the task who set up their RPA. So changing the major looks good to me.

Reply:
I think historically we've been pretty arbitrary on what version bump to do, so I wouldn't worry about it. New mandatory parameter -> major bump is pretty much the only rule I've tried to enforce

Reply:
I think we should get rid of versions and release notes :) I agree it doesn't matter much.